foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -62,3 +59,5 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 *   [This flexbox visual cheatsheet](https://flexbox.malven.co/) has some useful references to flex and its properties.
+
+* For an interactive demo of how axes work with flexbox, check out [this Scrim](https://scrimba.com/learn/flexbox/main-axis-and-cross-axis-flexbox-tutorial-cz94MT8).
```

Comment:
```suggestion
* For an interactive demo, check out this [Scrim on Flexbox axes](https://scrimba.com/learn/flexbox/main-axis-and-cross-axis-flexbox-tutorial-cz94MT8). Note that this Scrim requires logging into Scrimba in order to view.
```

I'm noticing the needing to login issue. Can you confirm whether you get that too, @SohamKukreti ?

Reply:
Yeah I get the login prompt as well. I suppose scrimba has change some of their policies recently